kolibri/core/assets/src/views/sync/FacilityNameAndSyncStatus/index.vue

```diff
@@ -111,6 +112,12 @@
         }
         return this.$formatRelative(datetime, { now: this.now });
       },
+      manageSync() {
```

Comment:
You mention this in your PR description:
> Added a click events in the common status component to avoid encoding facility specific navigation events in the core component .

But this is doing the exact opposite - it's removing the `manageSync` event emitting and instead encoding the route navigation here.